vmm/src/device_manager/mmio.rs

```diff
@@ -267,7 +270,8 @@ impl MMIODeviceManager {
     ) -> Option<&Mutex<BusDevice>> {
         if let Some(dev_info) = self
             .id_to_dev_info
-            .get(&(device_type, device_id.to_string()))
+            .iter()
+            .find(|&x| x.dev_type == device_type && x.dev_id == device_id.to_string())
```

Comment:
```suggestion
.find(|&x| x.dev_type == device_type && x.dev_id == device_id)
```
in order to get rid of the clippy error.